### PR TITLE
Sync `master` with `develop`

### PR DESCRIPTION
**Pull request by danjm**
This brings develop up-to-date with `v9.5.9`

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2260,6 +2267,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Uncategorized
 - Added the ability to restore accounts from seed words.
 
+[Unreleased]: https://github.com/MetaMask/metamask-extension/compare/v9.5.9...HEAD
+[9.5.9]: https://github.com/MetaMask/metamask-extension/compare/v9.5.8...v9.5.9
 [Unreleased]: https://github.com/MetaMask/metamask-extension/compare/v9.5.8...HEAD
```
Comment (Gudahtt) [inline CHANGELOG.md:2272]: Extra `Unreleased` entry

Comment (danjm) [reply CHANGELOG.md]: fixed in latest push

**Review by dan437 — APPROVED** [review]
It looks good to me

**Review by Gudahtt — APPROVED** [review]
LGTM!

**Pull request merged**